src/core/Object3D.d.ts

```diff
@@ -54,7 +55,7 @@ export class Object3D extends EventDispatcher {
 	 * Array with object's children.
 	 * @default []
 	 */
-	children: Object3D[];
+	children: Mesh[];
```

Comment:
I'm afraid this is not right. You assume here that only meshes can  be children of `OBject3D`.

Reply:
No i didn't think I would be, but I wanted to discuss it none the less & fix. Maybe `Object3D` should have a parameter `geometry?: BufferGeometry`? What do you think about this issue.

Reply:
I'm not using TS so I'm not sure. But since `Object3D.children` can hold any type of 3D objects (points, lines, groups, bones etc.) it's probably necessary to cast objects to a specific types if you want to access a geometry property.

Reply:
I think that makes sense. I suppose one day it might be good to infer the type 🤔. Thank you.